### PR TITLE
Fix: [MIQ-2130] Expose more vars and change some default values

### PR DESCRIPTION
**Pull request by shmsr**
### CATEGORY

Choose one

- [ ] Bug Fix
- [x] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [x] Build / Development Environment
- [ ] Documentation

### SUMMARY
Expose some of the relevant variables as well change some of the default values

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ superset_docker_restart_policy: "Always"
 superset_service_port: "32443"
 superset_nginx_port: "32443"
 
-SUPERSET_NPROCESSORS: 1
+SUPERSET_NPROCESSORS: 4
```
Comment (arpit-agarwal) [inline superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml]: We should relay on extra_vars for modifing these defaults. As on low footprint machine we would not like to increase the same

File: superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml [hunk 2]
```diff
@@ -120,22 +120,29 @@ SUPERSET_AUTH_LDAP_ALLOW_SELF_SIGNED: False
...
-RVF_APP_NAME: "Dashboard Builder"
+RVF_APP_NAME: "Dashboard"
 RVF_HELP_ENABLED: False
 SUPERSET_SESSION_LIFETIME_SECONDS: "3600" # stale session timeout
+RVF_LOG_FORMAT: "%(asctime)s:%(levelname)s:%(name)s:%(message)s"
 RVF_LOG_LEVEL: "INFO"
-RVF_CACHE_DEFAULT_TIMEOUT: "{{ 60 * 60 * 24 }}"
-RVF_CACHE_TYPE: 'null'
+RVF_CACHE_DEFAULT_TIMEOUT: "{{ 60*60*24 }}"
+RVF_CACHE_TYPE: 'simple' # enables simple caching to postgres
```
Comment (arpit-agarwal) [inline superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml]: cache is not enabled by default purposefully as it has some limitation that is counter intuitive. we will allow enabling this via extra_vars or configmap

File: superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml [hunk 3]
```diff
@@ -120,22 +120,29 @@ SUPERSET_AUTH_LDAP_ALLOW_SELF_SIGNED: False
...
 RVF_ENABLE_CORS: True
 RVF_CORS_OPTIONS: "{}"
 RVF_WALKME_ENABLED: True
+RVF_DEFAULT_RELATIVE_END_TIME: 'now'
```
Comment (arpit-agarwal) [inline superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml]: is this a new var introduced to take care of some use case?

**Review by arpit-agarwal — CHANGES_REQUESTED** [review]
Our installer has been moved  here
https://github.com/Guavus/raf-provisioner-composition/blob/master/playbooks/raf/raf_extra_vars.yml

File: superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml (hunk 1 above)
Comment (subham-sarkar-guavus) [reply superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml]: Reverted.

File: superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml (hunk 2 above)
Comment (subham-sarkar-guavus) [reply superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml]: Reverted.

File: superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml (hunk 3 above)
Comment (subham-sarkar-guavus) [reply superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml]: No, it's there already.

**Review by shmsr — COMMENTED** [review]
I've reverted back some of the changes.

**Pull request closed without merging**